service-catalog/components/grafana.yaml

```diff
@@ -3,6 +3,8 @@ kind: Component
 metadata:
   name: Grafana
   description: Dashboard anything. Observe everything
+  annotations:
+    operate-first.cloud/component-icon-url: https://upload.wikimedia.org/wikipedia/en/a/a1/Grafana_logo.svg
```

Comment:
We can't use icons from the website, because the idea is for them to go away at some point (once we don't need them on the website). I'll change this to https://raw.githubusercontent.com/grafana/grafana/main/public/img/grafana_icon.svg